.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "esbonio.sphinx.confDir": ""
+}
```

Comment:
VS Code also kept adding this for me, but can you remove this? I don't think it's really relevant